src/core/dbus-service.c

```diff
@@ -47,6 +47,59 @@ static BUS_DEFINE_PROPERTY_GET_ENUM(property_get_restart, service_restart, Servi
 static BUS_DEFINE_PROPERTY_GET_ENUM(property_get_notify_access, notify_access, NotifyAccess);
 static BUS_DEFINE_PROPERTY_GET_ENUM(property_get_emergency_action, emergency_action, EmergencyAction);
 
+static int property_get_exit_status_set(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+
+        ExitStatusSet *status_set = userdata;
+        Iterator i;
+        void *id;
+        int r;
+
+        assert(bus);
+        assert(reply);
+        assert(status_set);
+
+        r = sd_bus_message_open_container(reply, 'a', "s");
```

Comment:
hmm, i wonder what the right way to serialize this is, but I am pretty sure its not as a string, as that would be a cop-out from the dbus type system.

maybe we should serialize this as "a(ii)", where we'd basically serialize `.si_code` and `.si_status` fields of the `siginfo_t` structure `waitid()` returns. This would be inline with how we encode the exit code/status of individual commands in ExecCommand. I.e. for each member of `status_set->status` we'd return one `(ii)` consisting of CLD_EXITED and the value, and for each member of `status_set->signal` we'd return one `(ii)` consisting of CLD_KILLED and the value.